CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.19.0 (13/05/2024)
+
+**Common**
+* Added `Promote` to the `olp::cache::KeyValueCache` API to explicitly promote the key in the LRU cache.
```

Comment:
```suggestion
* Added `Promote` to the `olp::cache::KeyValueCache` API to explicitly promote the key in the LRU (Least Recently Used) cache.
```
It's always good to explain acronyms the first time they're used in the document.